src/libcore/macros.rs

```diff
@@ -168,8 +168,14 @@ macro_rules! try {
     })
 }
 
-/// Use the `format!` syntax to write data into a buffer of type `&mut Write`.
-/// See `std::fmt` for more information.
+/// Use the `format!` syntax to write data into a buffer.
+///
+/// The buffer must be a `&mut [Write][write]`.
```

Comment:
Whoa, links work like this? I figured because this was in a code block that links were disabled